app/views/api_page/_get_curr_weather.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class='weather-widget'>
+  <% params[:curr] ? get_weather(params[:curr],1) : get_weather(1) %>
```

Comment:
As we discussed the controller should give you the information you need in the view, and any other logic should go into POROs, models or services.